Radiant/fileWatcher.js

```diff
@@ -216,9 +218,17 @@ const uploadFile = function (info, endStream){
  * @returns {PromiseLike<T | T | never> | Promise<T | T | never>}
  */
 const makeCopy = function(source, destination) {
-    return fs.copyFile(source, destination).then(() => {
-        return fs.appendFile(destination, '#EXT-X-ENDLIST\n').then(() => {
-            return destination;
+    return new Promise((resolve, reject) => {
+        fs.copyFile(source, destination).then(() => {
+            // insert #EXT-X-START:TIME-OFFSET=0 at line 5 so we start at beginning and not from 'live'
+            // or most recent segment
+            insertLine(destination).content('#EXT-X-START:TIME-OFFSET=0').at(5).then((err) => {
```

Comment:
will this insert it many times or just once?

Reply:
each time the m3u8 has been modified .. segments added it will copy that m3u8 (which doesnt have the EXT-X-START) and then add that to the new copy.. I guess I could add that to the original m3u8 but the whole reason to copy it was so we dont run into any issues while FFMPEG is writing to it and we try to write to it

Reply:
ahhhh that's right that's right. Perfect! Thanks for the explanation.